autopush/router/apnsrouter.py

```diff
@@ -146,18 +147,22 @@ def _route(self, notification, router_data):
         try:
             apns_client.send(router_token=router_token, payload=payload,
                              apns_id=apns_id)
-        except ConnectionError:
+        except ConnectionError as ex:
             self.ap_settings.metrics.increment(
                 "updates.client.bridge.apns.connection_err",
                 self._base_tags
             )
+            self.log.error("Connection Error sending to APNS",
```

Comment:
ok there's 2 options to improve that:

log.failure("..", level=LogLevel.error) # sigh
log.error("..", log_failure=Failure()) # Failure grabs it from exc_info()